libpermian/plugins/anaconda_webui/__init__.py

```diff
@@ -460,18 +470,25 @@ def _clone_common(self):
         """ Clones repositories common to all Anaconda WebUI tests """
         self.log(f'Clonning common repositories')
         bots_dir = os.path.join(self.webui_dir, 'bots')
+        cockpit_common_dir = os.path.join(self.anaconda_dir, 'ui/webui/test/common')
 
-        # clone anaconda
-        self._clone_repo(self.git_anaconda_repo, self.git_anaconda_branch, self.anaconda_dir)
-
-        with tempfile.TemporaryDirectory() as temp_dir:
-            # clone cockpit/test/common
-            self._clone_repo(self.git_cockpit_repo, self.git_cockpit_branch, temp_dir)
-            shutil.move(os.path.join(temp_dir, 'test/common'),
-                        os.path.join(self.anaconda_dir, 'ui/webui/test/common'))
-
-        # clone bots
-        self._clone_repo(self.git_bots_repo, self.git_bots_branch, bots_dir)
+        if self.git_anaconda_repo.startswith('file://'):
+            # Create copy of anaconda repo
+            shutil.copytree(urlparse(self.git_anaconda_repo).path, self.anaconda_dir, symlinks=True, ignore_dangling_symlinks=True)
+        else:
+            # clone anaconda
+            self._clone_repo(self.git_anaconda_repo, self.git_anaconda_branch, self.anaconda_dir)
+
+        if not os.path.exists(cockpit_common_dir):
+            with tempfile.TemporaryDirectory() as temp_dir:
+                # clone cockpit/test/common
+                self._clone_repo(self.git_cockpit_repo, self.git_cockpit_branch, temp_dir)
+                shutil.move(os.path.join(temp_dir, 'test/common'),
+                            cockpit_common_dir)
+
+        if not bots_dir:
```

Comment:
It seems to me that `bots_dir` cannot be evaluated as False so this condition is never true???